.travis.yml

```diff
@@ -31,4 +31,4 @@ install:
     - sudo apt-get install -y subversion
 
 script: 
-    - rose test-battery --state=save -j 5 || (echo -e "\n\nRerunning Failed Tests...\n\n"; rose test-battery --state=save,failed -j 5)
+    - rose test-battery --state=save -j 5 || (echo -e "\n\nRerunning Failed Tests...\n\n"; rose test-battery -v --state=save,failed -j 5)
```

Comment:
Is the `-v` purposeful?

Reply:
Left for debug, probably still useful to capture future failures.